apps/web/pages/org/[id]/teams.tsx

```diff
@@ -1,8 +1,49 @@
 import { DashboardLayout } from "@app/layout";
+import { Teams } from "@app/views/dashboard"
 
 const teams = () => {
     return (
-        <div>teams</div>
+        <div className=" p-6">
+        <h1 className="text-xs">FOSSHACK 3.0</h1>
+        <div className="flex justify-between items-center ">
+          <h1 className="text-[38px] font-bold">Registrations</h1>
+          
+          <div className="flex">
+            <div className="items-center w-[auto] h-[auto] bg-brand-pink-100 rounded-lg p-2 mr-4">
+              <h1 className="text-[30px] text-center font-bold">124</h1>
```

Comment:
4px = 1 in tailwind border-3 border-[red]


so 38 px should be 9.5 ie make text-9.5